tekton/pipeline_run_test.go

```diff
@@ -0,0 +1,137 @@
+/*
+Copyright 2022 Red Hat Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package tekton
+
+import (
+	"context"
+	"reflect"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	"github.com/redhat-appstudio/release-service/api/v1alpha1"
+	tektonv1beta1 "github.com/tektoncd/pipeline/pkg/apis/pipeline/v1beta1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+type ExtraParams struct {
+	Name  string
+	Value tektonv1beta1.ArrayOrString
+}
+
+type ReleasePipelineParams struct {
+	Prefix    string
+	Namespace string
+	Extra     ExtraParams
+}
+
+var _ = Describe("PipelineRun", func() {
+
+	var params *ReleasePipelineParams
+	var releasePipelineRun *ReleasePipelineRun
+
+	var release *v1alpha1.Release
+	var releaseParams *v1alpha1.ReleaseSpec
+
+	BeforeEach(func() {
+
+		params = &ReleasePipelineParams{
+			Prefix:    "mypipeline",
+			Namespace: "default",
+			// @todo: use a valid Pipeline params
```

Comment:
Here are some examples. Any of them should work fine here
```
    - name: extraConfigGitUrl
      value: https://github.com/davidmogar/strategy-configs.git
    - name: extraConfigPath
      value: m5.yaml
    - name: extraConfigRevision # optional field, default is main
      value: main
```
if you pick the path one, please name it something besides mX.yaml (we are trying to avoid tying our names to milestones so we don't have to change stuff every milestone).